robocopkaka/bootcamp-29-project
# Feat create edit events component

#### What does this PR do?
Create component for editing events
#### Description of Task to be completed?
* Write tests for component
* Create component
* Create action and action creator for updating events
* Create static method for updating events
* Create case in event reducer that changes state when an event is successfully updated
* Create route for editing events in the `Main` component
* Add link for editing events to the EventsListWithImage component
#### How should this be manually tested?
* Clicking on the `edit` icon for any event should take you the edit page for that event.
*Change any info and click on `Update Event`
* You should be taken back to the event's show page with the updated info on display
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?(if applicable)
#155453851
